src/main/java/com/dev/cinema/model/Ticket.java

```diff
@@ -0,0 +1,27 @@
+package com.dev.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@NoArgsConstructor
+@Table(name = "ticket")
+public class Ticket {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @ManyToOne
+    private MovieSession movieSession;
+    @OneToOne
+    private User user;
+    @ManyToOne
+    private ShoppingCart shoppingCart;
```

Comment:
we don't need this relation here, ticket shouldn't know about shopping cart

Reply:
ok